var/spack/repos/builtin/packages/argobots/package.py

```diff
@@ -21,6 +21,7 @@ class Argobots(AutotoolsPackage):
     maintainers = ['shintaro-iwasaki']
 
     version("master", branch="master")
+    version("1.0", sha256="36a0815f7bf99900a9c9c1eef61ef9b3b76aa2cfc4594a304f6c8c3296da8def")
     version("1.0rc2", sha256="7496b8bd39930a548b01aa3b1fe8f8b582c272600ef6a05ddc4398cf21dc12a2")
```

Comment:
Can we remove the alpha/beta/rc versions not that a stable release is out?

Reply:
Currently some packages (`margo`, for example) point to a specific non-stable version.
https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/margo/package.py#L31 

I believe there is no compatibility issue, but can I remove the alpha/beta/rc versions once all the other packages use the version the stable 1.0 (or possibly newer)?

Reply:
Alright, we can wait then

Reply:
Thanks!